example-jasmine/tests/testCaseIdTest.spec.js

```diff
@@ -16,11 +16,20 @@
 
 const PublicReportingAPI = require('@reportportal/agent-js-jasmine/lib/publicReportingAPI');
 
-describe('testCaseId for suite/test', function() {
-    PublicReportingAPI.setTestCaseId('TestCaseIdForTheSuite', 'testCaseId for suite/test');
+describe('testCaseId for suite/test', function () {
+  PublicReportingAPI.setTestCaseId('TestCaseIdForTheSuite', 'testCaseId for suite/test');
 
-    it('should have the correct testCaseId', function() {
-        PublicReportingAPI.setTestCaseId('TestCaseIdForTheTest');
-        expect(true).toBe(true);
-    });
+  it('should have the correct testCaseId', function () {
+    PublicReportingAPI.setDescription(
+      'This test case contains automatically generated test case id',
+    );
+    PublicReportingAPI.addAttributes([
+      {
+        key: 'feature',
+        value: 'AutoGeneratedTestCaseId',
```

Comment:
Seems that here (in this test) you have an explicict `testCaseId`.
Please change the attrubute value to `ExplicitTestCaseId` and add another test with `AutoGeneratedTestCaseId` as [here](https://github.com/reportportal/examples-js/blob/master/example-cypress/cypress/integration/testCaseIdTest.spec.js).